### PR TITLE
AST-1373 - E2E test case for html2 font size

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for html2 font size

### Screenshots

https://share.getcloudapp.com/L1ueD8po

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/header-builder/html-2/html-2-font-size.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/html2/html2-font-size.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { createURL } from '@wordpress/e2e-test-utils';
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/html2/html2-font-size.test.js:1]: Please update path structure and file name:- tests/e2e/specs/customizer/header-builder/html-2/html-2-font-size.test.js

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/html2/html2-font-size.test.js]: Resolved

File: tests/e2e/specs/customizer/header-builder/html-2/html-2-font-size.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+import { setBrowserViewport } from '../../../../utils/set-browser-viewport';
+import { responsiveFontSize } from '../../../../utils/responsive-utils';
+describe( 'HTML 2 block settings in the customizer', () => {
+	it( 'html2 font size should apply correctly', async () => {
+		const htmlFontSize = {
+			'header-html-2': '<a href="https://wpastra.com/">HTML link color</a>',
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/html-2/html-2-font-size.test.js:8]:
```suggestion
			'header-html-2': '<a href="https://wpastra.com/">HTML font size</a>',
```

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/html-2/html-2-font-size.test.js]: Resolved

**Pull request merged**